components/fleet/FleetRepos.vue

```diff
@@ -28,6 +33,40 @@ export default {
   },
 
   computed: {
+    ...mapGetters(['isVirtualCluster']),
+    filteredRows() {
+      const isAll = this.$store.getters['isAllNamespaces'];
+      const isVirutalProduct = this.$store.getters['currentProduct'].name === HARVESTER;
+
+      // If the resources isn't namespaced or we want ALL of them, there's nothing to do.
+      if ( (!this.isNamespaced || isAll) && !isVirutalProduct) {
+        return this.rows || [];
+      }
+      const includedNamespaces = this.$store.getters['namespaces']();
```

Comment:
Git repos can't be created in a namespace, so why are we filtering by namespace?